src/Shouldly.Tests/ShouldBeLessThanOrEqualTo/CustomObjectScenario.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Shouldly.Tests.Strings;
+using Shouldly.Tests.TestHelpers;
+using Xunit;
+
+namespace Shouldly.Tests.ShouldBeLessThanOrEqualTo
+{
+    public class CustomObjectScenario
+    {
+        [Fact]
+        public void CustomObjectScenarioShouldFail()
+        {
+            var customA = new Custom { Val = 2 };
+            var customB = new Custom { Val = 1 };
+            var comparer = new CustomComparer<Custom>();
+            Verify.ShouldFail(() =>
+customA.ShouldBeLessThanOrEqualTo(customB, comparer, "Some additional context"),
+
+errorWithSource:
+@"customA
+    should be less than or equal to
+Shouldly.Tests.TestHelpers.Custom (000000)
+    but was
+Shouldly.Tests.TestHelpers.Custom (000000)
+
```

Comment:
Hi @ankurMalhotra, that's right - you can see a list of existing message generators [here](https://github.com/shouldly/shouldly/blob/master/src/Shouldly/ShouldlyMessage.cs#L213).

You're right, being a comparer you won't be able to get the value so just drop the value from the final message:

"customA should be less than or equal to customB but was not"

If you need any help with it or have any questions then feel free to let me know :)